include/LightGBM/config.h

```diff
@@ -353,6 +353,13 @@ struct Config {
   // desc = ``1`` means increasing, ``-1`` means decreasing, ``0`` means non-constraint
   // desc = you need to specify all features in order. For example, ``mc=-1,0,1`` means decreasing for 1st feature, non-constraint for 2nd feature and increasing for the 3rd feature
   std::vector<int8_t> monotone_constraints;
+
+  // type = multi-double
+  // alias = fc, fp, feature_penalty
+  // default = None
+  // desc = used to control feature's split gain, will use ``gain[i] = max(0, feature_contri[i]) * gain[i]`` to update split gain of i-th feature.
+  // desc = you need to specify all features in order.
```

Comment:
@guolinke Sorry for so insignificant comment, but please no dots at end of a line